platform-tests/src/platform/acceptance/mysql_service_test.go

```diff
@@ -16,7 +17,7 @@ import (
 var _ = Describe("MySQL backing service", func() {
 	const (
 		serviceName  = "mysql"
-		testPlanName = "Free"
+		testPlanName = "tiny-unencrypted-5.7"
```

Comment:
should this be the `tiny-5.7` plan instead otherwise won't it fail due to not being enabled?

Reply:
...same comment for the postgres one I guess

Reply:
fixed